vncviewer/org.tigervnc.vncviewer.metainfo.xml.in

```diff
@@ -42,4 +42,6 @@
       <image>https://raw.githubusercontent.com/TigerVNC/tigervnc/741d3edbfab65eda6f033078bc06347fe244ea6a/vncviewer/metainfo/tigervnc-connection-windows.jpg</image>
     </screenshot>
   </screenshots>
+  <developer_name>Pierre Ossman (Cendio AB) et al.</developer_name>
```

Comment:
This is not a one man show, so I'm afraid this is not a viable way forward. I think we should use the same "TigerVNC Team" we use in other places.

The AppStream specification does not list this as a required field, but I guess Debian has a stronger need for it?

Reply:
I have updated this but also kept the developer_name tag, as even in Debian Trixie, almost all meta info files still contain the developer_name tag, even if they have the new developer tag.

> The AppStream specification does not list this as a required field, but I guess Debian has a stronger need for it?

I got a warning about missing information from one of the many Debian policy checking tools.